frontend/src/stores/QueueStore.js

```diff
@@ -1,33 +1,101 @@
 import { observable, action, flow } from "mobx"
```

Comment:
@hhummel this is all looking fantastic, but the decorators are giving my linter a headache.  Take a look what I did here on the branch Robby and I are working on to get rid of it: https://github.com/CodeForPhilly/prevention-point/blob/Search_Participant_New/frontend/src/stores/ParticipantStore.js  The change was adding the `decorate` function to the bottom of the store.

Reply:
I remember seeing this discussed on mobx.  I wonder why I'm not having that problem

Reply:
its possible that you've turned it off on your linter or updated your linter.  But it definitely has issues with mine.  Do me a favor and in your local frontend directory do `yarn list eslint` and let me know what version of yarn you have

Reply:
~/projects/prevention-point/frontend$ yarn list eslint
yarn list v1.17.3
warning Filtering by arguments is deprecated. Please use the pattern option instead.
└─ eslint@5.16.0
✨  Done in 1.11s.
~/projects/prevention-point/frontend$ yarn list eslint
yarn list v1.17.3
warning Filtering by arguments is deprecated. Please use the pattern option instead.
└─ eslint@5.16.0
✨  Done in 1.11s.

Reply:
ok yeah that's the linter I have, so is it possible that you made changes to your linter file?